docs/en/template.md

```diff
@@ -1,13 +1,12 @@
 To automate deployment, this solution uses the following AWS CloudFormation templates, which you can download before deployment:
 
- [xxx.template](to be added): Use this template to launch the solution and all associated components. 
+ [quantum-ready-solution-for-drug-discovery.template](https://aws-gcr-solutions.s3.amazonaws.com/AWS-gcr-qc-life-science/v0.6.2/custom-domain/QCStack.template.json): Use this template to launch the solution and all associated components. 
```

Comment:
I have concern to put this beta version here.

Reply:
removed